python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -241,6 +241,7 @@ class GrapheneAssetNode(graphene.ObjectType):
     backfillPolicy = graphene.Field(GrapheneBackfillPolicy)
     changedReasons = graphene.Field(non_null_list(GrapheneAssetChangedReason))
     computeKind = graphene.String()
+    pools = non_null_list(graphene.String)
```

Comment:
nit: I think these fields are supposed to be in alphabetical order (not that we've been doing a great job of that) -- seems like there's no particular reason for it to be in this spot over it's alphabetical place so probably better to put it down near the partition stuff